src/PeselValidator/PeselValidator.cs

```diff
@@ -45,15 +45,12 @@ private static bool IsChecksumValid(string pesel)
         {
             int checksum = CalculateChecksum(pesel);
 
-            return pesel[10].ToString() == checksum.ToString();
+            return pesel.Last().ToString() == checksum.ToString();
         }
 
         private static int CalculateChecksum(string pesel)
         {
-            int sum = 0;
-
-            for (int i = 0; i < 10; i++)
-                sum += weights[i] * int.Parse(pesel[i].ToString());
+            var sum = weights.Select((x, i) => x * int.Parse(pesel[i].ToString())).Sum();
```

Comment:
Thanks @Jankowski-J for this PR. I checked the performance of this code and unfortunatly it is 3 times slower than the original one. 
I like the change with `Last()` though. It clrealy highlights the intent.
I did some more testing and looks like we can get the lost performance back by using `Convert.ToInt32` in line 53 and `Convert.ToChar` in line 48. Then these three `.ToString()`s are redundant and can be removed.
Could you please implement these changes?